### ISSUE TITLE
Can't get the results in the paper or the results of the pre-trained model provided

### ISSUE THREAD

**Issue by fanqinshan0114** [EricGuo5513/momask-codes#49]
I configured two different environments according to environment.yml and requirements.txt. Even if the seed is fixed, the training results in these two environments are inconsistent. why is that? And I still can't get the results in the paper or the results of the pre-trained model provided. Is this normal?

**Comment by Murrol** [comment]
First, the environments installed from the environment.yml and the requirements.txt are different. It's to help everyone run our model on different machines. For example, the pytorch versions are different. Version differences would affect the processed dataset and inference performance slightly. 

Second, I need to see your results to check if it's acceptable.
